src/EFCore.MySql/Infrastructure/CharSetAppendBehavior.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Pomelo Foundation. All rights reserved.
+// Licensed under the MIT. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Pomelo.EntityFrameworkCore.MySql.Infrastructure
+{
+    [Flags]
+    public enum CharSetAppendBehavior
```

Comment:
Did you consider dropping the `Append` as @Giantaxe suggested?

Reply:
Ok, I will simply call it `CharSetBehavior`